obspy/clients/fdsn/__init__.py

```diff
@@ -134,8 +134,69 @@
                                         endtime=endtime)
         inventory.plot()
 
+
+Basic FDSN FedCatalog Client Usage
+----------------------------------
+
+The
+:mod:`FDSN fedcatalog_client <obspy.clients.fdsn.routers.fedcatalog_client>`
+module provides federated
+access to multiple web servers that implement the
+`FDSN Station and Dataselect web service definitions
+<https://www.fdsn.org/webservices/>`_.
+
+The first step is always to initialize a :class:`FederatedClient` object.
```

Comment:
This link sees broken as well.. this should work, hopefully: ```:class:`~obspy.clients.fdsn.FederatedClient` ``` if not you have to specify full submodule path..